src/OSPSuite.Core/Domain/ModelCoreSimulationExtensions.cs

```diff
@@ -95,7 +95,7 @@ private static IEnumerable<IParameter> allApplicationParametersIn(IReadOnlyList<
             .Where(x => x.IsNamed(parameterName));
       }
 
-      public static void AddObjectSources(this IModelCoreSimulation simulation, EntitySources entitySources)
+      public static void AddEntitySources(this IModelCoreSimulation simulation, IEnumerable<EntitySource> entitySources)
```

Comment:
Do you really want IEnumerable here?

Reply:
I am not sure if I really want it but I figured since I am only interesting once and I do not know if the caller is going to have an Ienumerable, a cache or whatever, this is the most common interface.